documentation/src/routes/docs/[...8]comparisons/[...4]tRPC.md

```diff
@@ -6,12 +6,9 @@ Like Gravity, **tRPC** is very well typed.
 
 Advantages of tRPC over Gravity:
 
-- It uses [Zod](https://github.com/colinhacks/zod) to validate the client input types, so there is no need to generate a `schema.json` file like Gravity does.
-- Thanks to **Zod**, it is possible to validate complex types like emails, phone numbers, strings with a minimum length.
+- It uses validation libraries like [Zod](https://github.com/colinhacks/zod) to validate the client input types, so there is no need to generate a `schema.json` file like Gravity does.
 
 Advantages of Gravity over tRPC:
 
-- It does not use Zod, so defining server functions is **much more concise**.
-- tRPC use **string identifiers** to define and access queries. This kind of flat system does not allow namespaces and make it hard to scale. Instead, Gravity define proxies very easy to use that allow to define functions in **namespaces** that can be **infinitely nested**.
-- When using Prisma, Gravity allows **powerful declarative access** to the **database**.
```

Comment:
I'm talking about [Prisma constraints](https://gravity.digitak.dev/docs/prisma/prisma-service#constraints) which are a special Gravity feature when integrating with Prisma.

Reply:
Oh I really like that! Do you think we could abstract something like that so it could be used in tRPC? For the comparison, I think we should clarify that it's referring to the constraints.

Reply:
It can be definitely be a non-Gravity Prisma thing, and it would be much more cleaner if it had its own agnostic package.

For now I (sadly 😥) removed Prisma constraints from Gravity's documentation because it didn't integrate as well as expected with the types introspection system of Gravity. I will also remove this argument with the tRPC comparison.

Especially, if you have a model that has a relation with another model, the input type of the `prisma::create` function grows insanely huge, so huge that it can create an infinite-loop-like case when Gravity tries to parse the schema (not infinite but so big it's almost the same).

I'm ready to collaborate to bring the Prisma constraints to its own independent package if you're interested.